src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let cost = 40;
+
+  switch (true) {
+    case days >= 7
+      : cost = (cost * days) - 50;
```

Comment:
```suggestion
      : cost = (cost * days) - 50;
```
Вираз (cost * days) повторюється три рази. Давай винесемо його у змінну